### ISSUE TITLE
Recognise NixCon as a public communication resource

### ISSUE THREAD

**Issue by 7c6f434c** [nix-open-org/org-v2#22]
Not sure how exactly it should be worded, but NixCon is probably treated as official

**Comment by zimbatm** [comment]
Good idea. We could also have the @NixOS/org team take ownership of that org.

**Comment by infinisil** [comment]
@zimbatm Are you the only nixcon org owner and keyholder for nixcon.org?

**Comment by zimbatm** [comment]
Current "owners":
![image](https://github.com/user-attachments/assets/fd444899-955e-4544-bfa9-1a38ead1dd03)

**Comment by zimbatm** [comment]
I transferred the domain to the NixOS Foundation's Gandi account during my tenure there.

**Comment by infinisil** [comment]
Would be closed by https://github.com/NixOS/org/pull/74
